cluster/config-defaults.yaml

```diff
@@ -379,6 +379,9 @@ horizontal_pod_autoscaler_sync_period: "30s"
 horizontal_pod_autoscaler_tolerance: "0.1"
 horizontal_pod_downscale_stabilization: "5m0s"
 
+# Vertical pod autoscaler version config for controlling roll-out
+vertical_pod_autoscaler_version: "v0.11.0-internal.17"
```

Comment:
Maybe we can define this in a way so you don't need to know the old version to switch back. So instead of defining the version here we could have the value `current|legacy` or so and then have an if/else when choosing which image version to use.

Reply:
Hmm good idea. I'll add that!